File: go/delete_trunk.go
package main

import (
    "fmt"
    "os"
)

func main() {
    trunkSid := os.Getenv("TRUNK_SID")
    if trunkSid == "" {
        fmt.Println("Error: TRUNK_SID environment variable is required")
        os.Exit(1)
    }

    fmt.Printf("Deleting trunk %s...\n", trunkSid)
    result := delete(fmt.Sprintf("/trunks?trunk_sid=%s", trunkSid))
    fmt.Println("Trunk deleted successfully!")
    _ = result
} 

File: java/DeleteTrunk.java
public class DeleteTrunk {
    public static void main(String[] args) throws Exception {
        String trunkSid = System.getenv("TRUNK_SID");
        if (trunkSid == null || trunkSid.isEmpty()) {
            System.err.println("Error: TRUNK_SID environment variable is required");
            System.exit(1);
        }

        System.out.printf("Deleting trunk %s...%n", trunkSid);
        String result = _Client.delete(String.format("/trunks?trunk_sid=%s", trunkSid));
        System.out.println("Trunk deleted successfully!");
    }
} 